raw-mm.valid.main.lua

```diff
@@ -3397,7 +3394,7 @@ function valid.empty_pipe()
 end
 
 function valid.empty_light()
-  local r = checkany(dict.lightfaeleaf.physical, dict.lightmyrtle.physical, dict.lightcoltsfoot.physical, dict.lightsteam.physical)
+  local r = checkany(dict.lightsteam.physical)
```

Comment:
I believe there's a single-argument `checkaction` that would make more sense here now